<launch>
  <arg name="pcd_file" default="/home/GlobalMap.pcd" />

  <node name="pcl_publisher_node" pkg="pcl_publisher" type="pcd_filter_node" output="screen">
    <param name="input_pcd_file" value="$(arg pcd_file)" />
    <param name="leaf_size" value="0.01" />
    <param name="radius_search" value="0.5" />
    <param name="min_neighbors" value="10" />
    <!-- <param name="pass_through_min" value="10.0" />
    <param name="pass_through_max" value="-10.0" /> -->
  </node>
</launch>
